src/index.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
+import { BrowserRouter as Router } from "react-router-dom";
+import { ShortenUrlProvider } from "react-shorten-url";
+import { ModalProvider } from "./contexts/ModalProvider";
+import { AuthProvider } from "./contexts/AuthProvider";
+import { VideoProvider } from "./contexts/VideoProvider";
+import { LikeProvider } from "./contexts/LikeProvider";
+import { WatchlaterProvider } from "./contexts/WatchlaterProvider";
+import { PlaylistProvider } from "./contexts/PlaylistProvider";
+import { HistoryProvider } from "./contexts/HistoryProvider";
+
+const accessToken = process.env.REACT_APP_BITLY_ACCESS_TOKEN;
```

Comment:
I guess you should do that short link generation part on the backend. Nothing is hidden on the client-side so your BITLY_ACCESS_TOKEN is exposed now!